### PR TITLE
[BugFix] Add `sort` to tiingo equity historical

### PR DESCRIPTION
**Pull request by hjoaquim**
1. **Why**?:

    - Tiingo requires the `sort` argument. The endpoint breaks without it.

2. **What**? (1-3 sentences or a bullet point list):

    - Added the `sort` argument.

3. **Impact** (1-2 sentences or a bullet point list):

    - 5 (positive UX impact fixing an unusable endpoint)

4. **Testing Done**:

    - Run the endpoint for tiingo, before and after the fix.

### CODE REVIEW

File: openbb_platform/providers/tiingo/openbb_tiingo/models/equity_historical.py [hunk 1]
```diff
@@ -44,6 +44,24 @@ class TiingoEquityHistoricalQueryParams(EquityHistoricalQueryParams):
     _frequency: Literal["daily", "weekly", "monthly", "annually"] = PrivateAttr(
         default=None
     )
+    sort: Literal[
```
Comment (deeleeramone) [inline openbb_platform/providers/tiingo/openbb_tiingo/models/equity_historical.py:47]: Isn't this a fruitless effort to expose this param? Sorting occurs by date, ALWAYS for this endpoint, and results are always sorted in the same direction, ascending for every provider.

**Comment by deeleeramone** [comment]
I'm not able to recreate the issue you raise on the develop branch:

```
obb.equity.price.historical("AAPL", provider="tiingo").to_df()
```

Screenshot 2024-03-27 at 10 43 46 AM

File: openbb_platform/providers/tiingo/openbb_tiingo/models/equity_historical.py (hunk 1 above)
Comment (hjoaquim) [reply openbb_platform/providers/tiingo/openbb_tiingo/models/equity_historical.py:47]: I got this error without using the sort:
`"Error: Sort column name is not a valid column"`

So, apparently the `sort` needs to be explicit in the URL.

Comment (deeleeramone) [reply openbb_platform/providers/tiingo/openbb_tiingo/models/equity_historical.py:47]: How come it works for me right now, without this?

**Comment by hjoaquim** [comment]
Just figured that #6259 fixes this issue - the URL is built differently and then the `sort` is not mandatory (apparently). Not sure if we should keep it anyway. 
@deeleeramone @montezdesousa

File: openbb_platform/providers/tiingo/openbb_tiingo/models/equity_historical.py (hunk 1 above)
Comment (deeleeramone) [reply openbb_platform/providers/tiingo/openbb_tiingo/models/equity_historical.py:47]: The URL appears to work fine without a sort column.

```
https://api.tiingo.com/tiingo/daily/AAPL/prices?startDate=2023-03-27&endDate=2024-03-27&resampleFreq=monthly&token=
```

Screenshot 2024-03-27 at 10 54 01 AM

**Pull request closed without merging**